### PR TITLE
Request handler

### PR DESCRIPTION
**Pull request by carlitoswillis**
sorry if the changes seem excessive- just added eslint

the main edits were to the server- handling requests for specific entries by id.

### CODE REVIEW

File: database/dataGen.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ var genInsert = (number) => {
         // throw err;
       } else {
         if (number % 1000 === 0) {
-          console.log('error, ', number);
+          console.log('success, ', number);
```
Comment (jgenso1690) [inline database/dataGen.js:40]: please can you explain what is this checking?

Comment (carlitoswillis) [reply database/dataGen.js:40]: just to let me know theres progress during the data inserting

Comment (carlitoswillis) [reply database/dataGen.js:40]: but not too often, just every thousand entries

Comment (jgenso1690) [reply database/dataGen.js:40]: noted, thank you :+1:

**Pull request merged**